src/pudl/extract/eia930.py

```diff
@@ -47,40 +47,34 @@ def process_raw(
         return df.rename(columns=self.METADATA.get_column_map(page, **partition))
 
 
-raw_eia930__all_dfs = raw_df_factory(Extractor, name="eia930")
-
-
-@asset(
-    required_resource_keys={"datastore", "dataset_settings"},
-)
-def raw_eia930__balance(raw_eia930__all_dfs):
-    """Extract raw EIA 930 balance data from CSV sheets into dataframes.
-
-    Returns:
-        An extracted EIA 930 balance dataframe.
-    """
-    return Output(value=raw_eia930__all_dfs["balance"])
-
-
-@asset(
-    required_resource_keys={"datastore", "dataset_settings"},
-)
-def raw_eia930__interchange(raw_eia930__all_dfs):
-    """Extract raw EIA 930 interchange data from CSV sheets into dataframes.
+def raw_eia930_asset_factory(page: str):
+    """Asset factory for individual raw EIA 930 dataframes."""
+
+    @asset(
+        name=f"raw_eia930__{page}",
+        op_tags={"memory-use": "high"},
+        retry_policy=RetryPolicy(
+            max_retries=3,
+            delay=60,  # 1 minute
+            backoff=Backoff.EXPONENTIAL,
+            jitter=Jitter.PLUS_MINUS,
+        ),
+        compute_kind="pandas",
+    )
+    def _extract_raw_eia930(
```

Comment:
Replaces 3 individual assets using the same pattern with an asset factory (though the diff is a bit messy).

Also, tag these 3 raw EIA-930 as high memory since they can very briefly use 9-8GB of memory, and they seem to be running around the same time as the FERC-714 asset that is crashing out the nightly builds, and there are 3 of them that all run simultaneously.